core/blockly.js

```diff
@@ -276,7 +276,16 @@ Blockly.onKeyDown_ = function(e) {
     }
   } else if (Blockly.keyboardAccessibilityMode_
       && (e.keyCode === goog.events.KeyCodes.UP
-        || e.keyCode === goog.events.KeyCodes.DOWN)) {
+        || e.keyCode === goog.events.KeyCodes.DOWN
```

Comment:
A more common pattern for this is to return a "handled" boolean from the called method (`navigate()`, in this case).  In that way, you don't have to sync to locations of used keys. It does mean, if there is key mapping overlap, the function can override the main assignment.


```
   if (Blockly.keyboardAccessibilityMode_ && Blockly.Navigation.navigate(e)) {
     // return? maybe?  This block is really here to skip the following if else conditions.
   } else if ...